lib/emis/configuration.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+require 'common/client/configuration/soap'
+
+module EMIS
+  class Configuration < Common::Client::Configuration::SOAP
+    SSL_CERT = begin
```

Comment:
Between HCA, MVI and EMIS, we could pull this SSL_CERT/SSL_KEY load code into the SOAP class.